cwms-data-api/src/main/java/cwms/cda/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -881,8 +936,24 @@ private void store(Connection connection, String officeId, String tsId, String u
             }
         }
 
+        if(versionDate != null) {
+            try {
+                CWMS_TS_PACKAGE.call_SET_TSID_VERSIONED(getDslContext(connection, officeId).configuration(),
+                        tsId, "T", officeId);
+            } catch(DataAccessException e) {
+                SQLException cause = (SQLException)e.getCause();
```

Comment:
Need to `instasnceof` check to make sure the cause is actually a `SQLException`

Reply:
Done